addons/website_sale/tests/test_website_sale_pricelist.py

```diff
@@ -496,9 +496,6 @@ def setUp(self):
         Website = self.env['website']
         self.website = self.env.ref('website.default_website')
         self.website.company_id = self.company2
-        # Delete unused website, it will make PL manipulation easier, avoiding
-        # UserError being thrown when a website wouldn't have any PL left.
-        Website.search([('id', '!=', self.website.id)]).unlink()
```

Comment:
Same as other point, this is probably to avoid race condition and/or nightly errors when module install differ from regular build.
Is there any issue with this?